contracts/protocol/NonUpgradable/ProtocolModule.sol

```diff
@@ -12,14 +12,21 @@ import "./../Interfaces/IMetadataModule.sol";
 import './../Interfaces/ILegalContractMetadata.sol';
 import './../Interfaces/IMetadataRenderer.sol';
 import '@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol';
+import '@openzeppelin/contracts/utils/ReentrancyGuard.sol';
 
 
-contract ProtocolModule is Ownable, Pausable {
+contract ProtocolModule is Ownable, Pausable, ReentrancyGuard {
     using SafeERC20 for IERC20;
     
     uint256 public wrappedSongCreationFee;
     uint256 public releaseFee;
-    
+    uint256 public distributorCreationFee;
+    uint256 public updateMetadataFee;
```

Comment:
This block can be changed for a fees dictionary to avoid declaring multiple variables